frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable.tsx

```diff
@@ -0,0 +1,162 @@
+import { useCallback, useEffect, useMemo } from 'react'
+import { useNavigate } from 'react-router-dom'
+import {
+  Column,
+  useFlexLayout,
+  usePagination,
+  useResizeColumns,
+  useTable,
+} from 'react-table'
+import { Flex, Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react'
+
+import { StorageModeSubmissionMetadata } from '~shared/types'
+
+import { useUnlockedResponses } from './UnlockedResponsesProvider'
+
+type ResponseColumnData = StorageModeSubmissionMetadata
+
+const RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
+  {
+    Header: '#',
```

Comment:
Noticed that the # is in decreasing order in the table, is this supposed to represent the n-th response submission, that's why it's in reverse order? (since it's ordered by most recent submission time)

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/24553546/174715742-d0d82d37-6788-4a30-8f04-af4ca4a7beae.png">

Reply:
yes. and also its just what the backend returns hehe